hack_loki.md

```diff
@@ -0,0 +1,94 @@
+# Hacks on Loki & Grafana
+
+## Loki Operator
+
+### From Sources
+
+Clone loki-operator repository and open folder
+```bash
+git clone https://github.com/ViaQ/loki-operator.git
+cd loki-operator
+```
+
+Disable loki auth for development in [loki-config.yaml](https://github.com/ViaQ/loki-operator/blob/master/internal/manifests/internal/config/loki-config.yaml#L2)
+`sed -i 's/auth_enabled: true/auth_enabled: false/g' /internal/manifests/internal/config/loki-config.yaml`
+
+Build and push the container image and then deploy the operator
+`make oci-build oci-push deploy REGISTRY_ORG=netobserv VERSION=latest`
+
+Create a LokiStack instance to get the various components of Loki up and running
+`kubectl apply -f hack/lokistack_dev.yaml`
+
+### From image (WARNING WIP, need kubectl [1.23.0-alpha.4](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md#v1230-alpha4) or more. Issues on /host access in debug ephemeral container)
+
+Get operator from official image and create instance
+```bash
+kubectl apply -f examples/lokioperator.yaml
+kubectl apply -f examples/lokistack_dev.yaml
+```
+
+Get controller manager name
+`kubectl get pods -l name=loki-operator-controller-manager`
+
+Create ephemeral container using controller manager name like :
+`kubectl debug controller-manager-9cb578d85-5dcxg -it --target=manager --image=busybox`
+
+Disable auth in loki config 
+`sed -i 's/auth_enabled: true/auth_enabled: false/g' /host/workspace/internal/manifests/internal/config/loki-config.yaml`
+
+Then restart all loki pods to update configmap
+`kubectl get pods -n default --no-headers=true | awk '/loki-/{print $1}'| xargs  kubectl delete -n default pod`
+
+## Loki & Grafana stack with Helm
+
+Deploy [loki and grafana](https://grafana.com/docs/loki/latest/installation/helm/#deploy-grafana-to-your-cluster) to your cluster from official images using helm
+```bash
+helm upgrade --install loki grafana/loki-stack --set promtail.enabled=false
+helm install loki-grafana grafana/grafana
+```
+
+Wait for the service running then forward port
+`kubectl port-forward service/loki-grafana 3000:3000`
+
+Login : `admin`
+Password should be retreive with :
+`kubectl get secret loki-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo`
+
+### Troubleshooting
+If loki-grafana deployment fails with following error on OpenShift:
+Error creating: pods "loki-grafana-XXXXXXXXXX" is forbidden: unable to validate against any security context
+
+Simply run:
+`oc adm policy add-scc-to-group anyuid system:authenticated`
```

Comment:
this is giving very broad permissions to the `system:authenticated` group, while it can be ok as long as it's clearly in a "hacks on loki" doc and nothing official, probably we should take care about not advertising this kind of broad solution.

I just did a try with this SCC given just to the loki and grafana service accounts, and it seems to work as well:

```bash
oc adm policy add-scc-to-user anyuid -z loki
oc adm policy add-scc-to-user anyuid -z loki-grafana
```

Reply:
Looks good to me :+1: 

@memodi did you try that ?

Reply:
@jotak you're right, this gives broad permissions and has overall security implications on cluster. I'll try today with -z option.

Reply:
confirming -z options works.